packages/beacon-node/src/network/peers/discover.ts

```diff
@@ -402,7 +402,10 @@ export class PeerDiscovery {
     const syncnetsBytes = enr.kvs.get(ENRKey.syncnets); // 4 bits
     const custodyGroupCountBytes = enr.kvs.get(ENRKey.cgc); // 64 bits
     if (custodyGroupCountBytes === undefined) {
-      this.logger.warn("peer discovered with no cgc assuming 4", exportENRToJSON(enr));
+      this.logger.warn(
```

Comment:
we should consider moving this to `debug`, I don't see how a user can do anything with the log and for us when debugging with have all log levels enabled anyways, so no difference